c++/src/capnp/compiler/error-reporter.h

```diff
@@ -54,6 +54,8 @@ class ErrorReporter {
   // is to inhibit the reporting of errors which may have been caused by previous errors, or to
   // allow the compiler to bail out entirely if it gets confused and thinks this could be because
   // of previous errors.
+
+  virtual void reportResolution(uint32_t startByte, uint32_t endByte, uint64_t typeId){};
```

Comment:
Perhaps this should take `Resolution` as its parameter?

Reply:
I agree that. I will replace with `Resolution`.   
In addition, I will move declaration of `Resolution` to `error-reporter.h`.